### PR TITLE
Update the setup.py for the 0.2.0 release

### PR DESCRIPTION
**Pull request by jeremycline**
Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

**Comment by pypingou** [comment]
👍 for me

File: setup.py [hunk 1]
```diff
@@ -27,13 +27,13 @@ def get_requirements(filename='requirements.txt'):
 
 setup(
     name='fmn.sse',
-    version='0.1.1',
+    version='0.2.0',
     description='Real time feed for fedmsg',
     long_description=get_description(),
-    author='Szymon M',
-    author_email='skrzepto@gmail.com',
+    author='Fedora Infrastructure Team and Szymon M',
```
Comment (pypingou) [inline setup.py:33]: Maybe something like ``Szymon M, Jeremy Cline and the Fedora Infrastructure Team`` ? Or we could use the maintainer field.

Just food for thoughts here, I don't think it really matters

Comment (jeremycline) [reply setup.py:33]: Yeah. I don't have strong feelings one way or the other, so I'm inclined to leave it as is. I think I'd prefer to maintain an AUTHORS file and include that in the packaging.

Comment (pypingou) [reply setup.py:33]: Up to you :)

I maintain my 👍

**Pull request merged**